### ISSUE TITLE
Isometric (Staggered) maps are not supported

### ISSUE THREAD

**Issue by adrien-bon** [adrien-bon/bevy_ecs_tiled#31]
When loading an isometric "staggered" map, tiles do not render properly (issue with `bevy_ecs_tilemap` or the way we use it ?) and objects placement is messed up.
See 2nd map of `examples/iso_map.rs`

Since it looks like to be not so easy to fix (not well supported by `bevy_ecs_tilemap` ? multiple combinations of stagger axis/stagger index) and we already have the isometric "diamond" map with the same characteristics, I won't fix this for now.

I added a warning when an user tries to load such a map so he is warned!

**Comment by darkautism** [comment]
Is the collider spawn in different space is the same bug?
![image](https://github.com/user-attachments/assets/d66e26ae-73e9-40f7-aba2-d45dd4f7f1a0)

**Comment by darkautism** [comment]
Hex has the same bug
![image](https://github.com/user-attachments/assets/36994421-60ff-48df-ac18-5948f6a7cf63)

![image](https://github.com/user-attachments/assets/58667caf-12a2-48db-8d1d-a60dfd1876a2)

![image](https://github.com/user-attachments/assets/95394457-0bca-4a35-bab0-55b54b662072)

**Comment by adrien-bon** [comment]
No, I don't think it's the same bug. This issue is to track the support for "staggered" isometric map, which currently do not render properly (tiles are all messed up). In your case, the render seems to be correct.

What you observe looks like to be an issue with colliders and objects position.
It's a different thing.

I bet you are using "infinite" maps: I know they are not well supported right now for isometric and hexagonal orientations, but I did not do extensive testings.
Can you please confirm you are using an infinite map in you screenshot ?
If so, I'll open a dedicated issue to track this bug.
